Lib/test/test_cprofile.py

```diff
@@ -25,7 +25,6 @@ def test_bad_counter_during_dealloc(self):
         with support.catch_unraisable_exception() as cm:
             obj = _lsprof.Profiler(lambda: int)
             obj.enable()
-            obj = _lsprof.Profiler(1)
```

Comment:
Why has this line been removed?

Reply:
Because what the test was partially testing won't work anymore. The line relies on the fact that the original obj was deallocated when replace - that won't happen because it's still being referenced by the monitors.

Having that line would also cause an infinite exception loop, probably due to the new mechanism.